src/NexusMods.App.UI/Pages/Settings/SettingsPageViewModel.cs

```diff
@@ -49,6 +46,21 @@ public SettingsPageViewModel(ISettingsManager settingsManager, IWindowManager wi
             }
         }, this.WhenAnyValue(vm => vm.HasAnyValueChanged));
 
+        CancelCommand = ReactiveCommand.Create(() =>
+        {
+            // TODO: ask to discard current values
+            foreach (var viewModel in SettingEntries)
+            {
+                viewModel.InteractionControlViewModel.ValueContainer.ResetToPrevious();
+            }
```

Comment:
This doesn't seem to update the selected item in the Dropdown for severity or the langue setting